logging.md

```diff
@@ -0,0 +1,359 @@
+Actinia supports logging to fluentd as well as to stdout.
+
+## tl;dr:
+
+* Rename logfile in actinia.cfg in project setups from
```

Comment:
```suggestion
* Rename logfile in actinia.cfg in project setups from
```

... I suppose this only refers to already existing actinia.cfg files?

Reply:
Right, if in certain project setups another config is used than the default config.